### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by barryjcooks**
update the README to reflect the work on values launched in 2020.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,34 @@ For some context on why we think having a code of conduct for our organization i
 
 # DigitalOcean's Engineering Code of Conduct
 
+At DigitalOcean we strive to be a values driven company.
```
Comment (vaskoz) [inline README.md]: Grammarly suggestion: "values driven" to "values-driven".

File: README.md [hunk 2]
```diff
@@ -2,8 +2,34 @@ For some context on why we think having a code of conduct for our organization i
...
+
+First lets look at those values
```
Comment (vaskoz) [inline README.md]: "First lets" => "First, let's"

File: README.md [hunk 3]
```diff
@@ -2,8 +2,34 @@ For some context on why we think having a code of conduct for our organization i
...
+
 ## Why have a Code of Conduct?
-This Code of Conduct is designed to help all of us build a pleasant, productive, and fearless community. The purpose of the Code of Conduct is not to burden the team with a bunch of needless rules, or to give us a punishment mechanism for people "being bad," or even to correct things that have been wrong in the past. We are striving to make our engineering team a great group of people to work with, especially for those people who have faced more adverse working environments in the past.
+This Ccode of Conduct is designed to help provide a translation between the values we aspire to and the mechanics of how to achive them in practice. The Code of Conduct is designed to help all of us build a pleasant, productive, and fearless community. The purpose of the Code of Conduct is not to burden the team with a bunch of needless rules, or to give us a punishment mechanism for people "being bad," or even to correct things that have been wrong in the past. We are striving to make our engineering team a great group of people to work with, especially for those people who have faced more adverse working environments in the past.
```
Comment (vaskoz) [inline README.md]: 2 typos: "Ccode", "achive"

File: README.md [hunk 4]
```diff
@@ -29,6 +55,10 @@ Denote [working hours](https://support.google.com/calendar/answer/83117?hl=en#wo
 Resist the temptation to have local-only meetings when remote teammates should be involved - take them to Slack, Hangouts, etc. as appropriate.
 
 ## Giving and Receiving Feedback
+
+### Assume positive intent
+All too often the first reaction to feedback or questions can be negative based on the assumption that the intent of the feedback was to critize or derail your idea. Maybe the phrasing wasn't ideal but was the feedback valuable when you assume they are trying to help make you or your work better?
```
Comment (vaskoz) [inline README.md]: typo: "critize"

**Review by vaskoz — COMMENTED** [review]
Hi @barryjcooks , I found a few small suggestions for your contribution. Please review them at your convenience.

File: README.md (hunk 1 above)
Comment (barryjcooks) [reply README.md]: Agree. fixing that

File: README.md (hunk 2 above)
Comment (barryjcooks) [reply README.md]: On additional reading I think the line is extraneous so I removed it altogether.

File: README.md (hunk 3 above)
Comment (barryjcooks) [reply README.md]: thanks... fixed.

File: README.md (hunk 4 above)
Comment (barryjcooks) [reply README.md]: nice catch.

**Pull request closed without merging**